staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/validation/validation.go

```diff
@@ -77,6 +77,9 @@ func ValidateLabels(labels map[string]string, fldPath *field.Path) field.ErrorLi
 
 func ValidateDeleteOptions(options *metav1.DeleteOptions) field.ErrorList {
 	allErrs := field.ErrorList{}
+	if options.GracePeriodSeconds != nil && *options.GracePeriodSeconds < 0 {
```

Comment:
@deads2k Thanks for your feedback. Do you mean I should move the grace period validation to some other place, or we just do not necessarily need this check?